tox.ini

```diff
@@ -26,7 +26,6 @@ envlist =
 
     ; opentelemetry-exporter-jaeger
     py3{5,6,7,8,9}-test-exporter-jaeger
-    pypy3-test-exporter-jaeger
```

Comment:
Why is this being removed?

Reply:
https://github.com/open-telemetry/opentelemetry-python/blob/master/tox.ini#L31-37

I think pypy3 doesn't work with exporters which have protobuf via gRPC. It is excluded from `otlp` and `opencensus`. It was failing for jaeger as well, that's why I had removed this.